rotkehlchen/assets/converters.py

```diff
@@ -827,11 +832,17 @@ def asset_from_coinbase(cb_name: str, time: Optional[Timestamp] = None) -> Asset
         if time < COINBASE_DAI_UPGRADE_END_TS:
             # Then it should be the single collateral version
             return A_SAI
-        # else
         return A_DAI
 
-    # else
-    return symbol_to_asset_or_token(cb_name)
+    if not isinstance(cb_name, str):
+        raise DeserializationError(f'Got non-string type {type(cb_name)} for coinbase asset')
```

Comment:
There is no need for this. If a non-string ends up in here it's an assertion error, not a deserialization error. Should have been picked up much earlier.

I notice that it's all in the other function so don't remove it. I am making a mental note to check if it can be removed from all of them.